lib/Configuration.py

```diff
@@ -1100,9 +1100,37 @@ def FindLatestManifest(self, train=None, require_signature=False):
             else:
                 train = temp_mani.Train()
 
+        if enterprise:
+            mani_file = self.TryGetNetworkFile(url="%s/%s/LATEST-ENTERPRISE" % (self.UpdateServerMaster(), train),
+                                               reason="GetLatestManifest")
+            if mani_file is None:
+                log.debug("Could not get latest enterprise manifest file for train %s" % train)
+                return None
+
+            rv = Manifest.Manifest(self, require_signature=require_signature)
+            rv.LoadFile(mani_file)
+            mani_file.close()
+
+            update_rate = rv.dict().get("rate")
+            # update_rate = 0: LATEST version is not enterprise-ready, always using LATEST-ENTERPRISE version
```

Comment:
Is it easy to expand this a little? Also have rate for general TrueNAS? SO we can control how many people we offer upgrades?

Reply:
@william-gr so we'll have LATEST-ENTERPRISE and LATEST-CORE files? (and, if they permit, request LATEST)

Or we better have different LATEST for CORE and ENTERPRISE?

Reply:
Is it possible to have CORE use LATEST and still be backward compatible? So we avoid one extra network call